src/store/modules/app/actions.js

```diff
@@ -40,10 +45,14 @@ export default {
         const param = queryParamsToHandle[key]
 
         commit(param.mutationName, query[key], param.mutationConfiguration)
+
+        if (param.clearUserState) {
+          commit('auth/CLEAR_USER_STATE', null, { root: true })
+        }
       }
     })
 
-    router.replace(rootState.route.name)
```

Comment:
This was the original bug. When you pass a string to `router.replace()`, it treats it as a path. Changing this to `router.replace({ name: rootState.route.name })` fixed the original "refresh on record detail page" bug.

That being said, I think we should be explicit here:

```javascript
router.replace({ path: rootState.route.path })
```